src/Service/EloquaApiClient.php

```diff
@@ -150,7 +162,7 @@ public function getAccessTokenByRefreshToken() {
    *   a new access token, token type, access token expiration time, and
    *   new refresh token
    */
-  private function doTokenRequest(array $params, $res = 'token') {
+  public function doTokenRequest(array $params, $res = 'token') {
```

Comment:
This was changed deliberately so that `doTokenRequest` can be reused in drush service

Reply:
private to public makes sense :)